### DISCUSSION TITLE
Pygmalion-6b problem

### DISCUSSION THREAD

**Discussion by deng232** [KoboldAI/KoboldAI-Client#366] [Q&A]
I got 
```You are using a model of type gptj to instantiate a model of type gpt_neo. This is not supported for all configurations of models and can yield errors.``` 
when loading Pygmalion-6b model; and the loading fail with few more errors.

It seems like it related to configuration, any idea how to fix it?

(I'm running on RTX 3080 TI, Win11)

**Comment by henk717** [comment]
The real error is further up, but its likely a VRAM limitation. If you are on KoboldAI United try turning 4-bit on at the bottom of the sliders.

**Reply by deng232** [reply]
The error is in KoboldAI-Client, I got it running in your KoboldAI United(both 4 bit and 8bit with cache in CPU).

from the stack trace of KoboldAI-Client, it didn't raise any runtime error about VRAM though
